src/cf-modals/src/cf-modals.less

```diff
@@ -0,0 +1,91 @@
+/* ==========================================================================
+   Capital Framework
+   Modal Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// .o-modal
+@modal-backdrop:   @black;
+@modal-body:       @white;
+@modal-footer:     @gray-5;
+@modal-border-top: @green;
+
+// Import external dependencies
+
+@import (less) '../../cf-core/src/cf-core.less';
+@import (less) '../../cf-icons/src/cf-icons.less';
+
+.o-modal {
+    z-index: 99999;
+    display: none;
+
+    &__visible {
+        display:block;
+    }
+
+    &_instructions {
+      .u-visually-hidden();
+    }
+
+    &_backdrop {
+        position: fixed;
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        background-color: @modal-backdrop;
+        opacity: 0.5;
```

Comment:
opacity should be set to 0.7 according to Lorelei's DM updates: https://ielerol.github.io/design-manual/page-components/modals.html#style

Reply:
Fixed in https://github.com/cfpb/capital-framework/pull/583/commits/d1674b14e2f9f7262c8cbbef6fa13001bf2de7a6